packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -568,15 +564,21 @@ export function signInActor({ services }: SignInMachineOptions) {
           groupLog('+++shouldRedirectToConfirmSignUp', 'event', event);
           return event.data.nextStep?.signInStep === 'CONFIRM_SIGN_UP';
         },
-        shouldRequestVerification: (_, event): boolean => {
-          groupLog('+++shouldRequestVerification', 'event', event);
+        shouldRequestVerification: (context, event): boolean => {
+          groupLog('+++shouldRequestVerification', context, event);
           const { phone_number_verified, email_verified } =
             event.data as Auth.FetchUserAttributesOutput;
 
-          // @todo-migration only request verificaion if phone or email is not verified and it is required
-          return false;
+          // email/phone_verified is returned as a string
+          const emailRequiredAndNotVerified =
+            !isEmpty(email_verified) && email_verified === 'false';
+          const phoneRequiredAndNotVerified =
+            !isEmpty(phone_number_verified) &&
+            phone_number_verified === 'false';
```

Comment:
Curious why we need to inspect the contents `email_verified`/`phone_number_verified` and add the conditional checks?

Reply:
`phone_number_verified` and `email_verified` are not always returned so I wanted to ensure that they were explicitly set to false.

Reply:
Ahhh that makes sense. Tricky truthy strings 😆